constitution.md

```diff
@@ -0,0 +1,684 @@
+
+_AdmNot-1_: For the purposes of this constitution, a notice may be given to or served on a person:
+
+* by delivering the notice to the person personally, or
+* by sending the notice by pre-paid post to the address of the person, or
+* by sending the notice by electronic transmission to an address specified by the person for giving or serving the notice.
+
+_AdmNot-2_: A notice is taken to have been given to or served on a person, unless the contrary is proved:
+
+* for a notice given or served personally - on the date on which the notice is received by the person, or
+* for a notice sent by pre-paid post - on the date on which the notice would have been delivered in the ordinary course of post, or
+* for a notice sent by electronic transmission:
+  - on the date the notice was sent, or
+  - if the machine from which the transmission was sent produces a report indicating the notice was sent on a later date—on the later date.
+
+### _AdmCus_: Custody of records and books
+
+_AdmCus-1_: Except as otherwise provided by this constitution, all records, books and other documents relating to the association must be kept in New South Wales:
+
+* at the association’s main premises, in the custody of either of the following persons, as determined by the committee:
+  - the public officer,
+  - a member of the association, or
+* if the association has no premises - at the association’s official address, in the custody of the public officer.
+
+### _AdmIns_: Inspection of records and books
+
+_AdmIns-1_: The following documents must be available for inspection, free of charge, by members of the association at a reasonable time:
+
+* this constitution,
+* minutes of committee meetings and general meetings of the association,
+* records, books and other documents relating to the association.
+
+_AdmIns-2_: A member may inspect a document referred to in subclause _AdmIns-1_:
+
+* in hard copy, or
+* in electronic form, if available.
+
+_AdmIns-3_: A member may obtain a hard copy of a document referred to in subclause _AdmIns-1_ on payment of a fee of not more than $1, as determined by the committee, for each page copied.
+
+_AdmIns-4_: The committee may refuse to allow a member to inspect or obtain a copy of a document under this clause:
+
+* that relates to confidential, personal, commercial, employment or legal matters, or
+* if the committee considers it would be prejudicial to the interests of the association for the member to do so.
+
+### _AdmFin_: Financial year
+
+_AdmFin-1_: The association’s financial year is:
+
+* The period commencing on the date of incorporation of the association and ending on the following 30 June, and
+* each period of 12 months after the expiration of the previous financial year, commencing on 1 October and ending on the following 30 September.
+
+Note: The Regulation, section 21 contains a substitute clause 44 for certain associations incorporated under the Associations Incorporation Act 1984.
```

Comment:
❓ Does this apply? We probably want to either remove this line, or use the substitute clause, depending on whether it does apply.

Reply:
Looks like it doesn't apply as there was a "Associations Incorporation Act 1985", and LA was incorporated long after that.  I'll remove it.

Reply:
For the record: I checked the Regulation, and rather than being anything important, section 21 just provides transitional arrangements for pre-2022 associations to have more flexibility in their financial year if their constitution doesn't cover it; since we're choosing a specific arrangement here it isn't relevant.